.github/workflows/cmake.yml

```diff
@@ -0,0 +1,360 @@
+name: Build
+
+on: [push]
+
+jobs:
+  linux-gcc:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        VER: [7, 8, 9]
+        EXT: [ON, OFF]
+        GEN: [Unix Makefiles, Ninja]
+        VAR: [Debug, Release]
+        STD: [11] # 90 results in errors
```

Comment:
What about 99? We should probably test C++ as well. I'm not saying this should be in scope for this PR, just that we should track/discuss it.

Reply:
Added.

Reply:
That's good enough for a first step, resolving.